test/scripts/imgtestlib.py

```diff
@@ -362,12 +369,24 @@ def read_osrelease():
 
 
 def get_osbuild_commit(distro_version):
+    """
+    Get the osbuild commit defined in the Schutzfile for the host distro.
+    If not set, returns None.
+    """
     with open(SCHUTZFILE, encoding="utf-8") as schutzfile:
         data = json.load(schutzfile)
 
     return data.get(distro_version, {}).get("dependencies", {}).get("osbuild", {}).get("commit", None)
 
 
+def get_osbuild_nevra():
+    """
+    Returned the installed osbuild version. Exits with an error if osbuild is not installed.
+    """
+    out, _ = runcmd(["rpm", "-q", "osbuild"])
```

Comment:
I have a super-nitpick! TIL I learned that `rpm -q` doesn't return the epoch. So we should either rename this function to `get_osbuild_nvra`, or use a different `rpm` invocation, see:

```
$ rpm -q --qf %{nevra} podman
podman-5:4.9.0-1.fc39.x86_64
$ rpm -q  podman
podman-4.9.0-1.fc39.x86_64
```

Reply:
Fixed